Project.toml

```diff
@@ -1,29 +1,30 @@
 name = "AllocArrays"
 uuid = "5c00bae2-1499-4716-9206-27f63fd08a44"
 authors = ["Eric P. Hanson"]
-version = "0.4.0"
+version = "0.4.1"
 
 [deps]
+Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
```

Comment:
Isn't this a good fit for an extension?

Reply:
good point, it's perfect for that. I'll update this tonight